Automattic/jetpack
# Random Redirect: add module to `jetpack-mu-wpcom` package

## Proposed changes:

The Random Redirect module was removed from the Jetpack plugin in https://github.com/Automattic/jetpack/pull/38310. However, some simple sites still relied on it.

This PR adds the Random Redirect module to the `jetpack-mu-wpcom` package to prevent those sites from breaking.

### Other information:


## Jetpack product discussion


p1721192843876849-slack-C043B54G6V7

## Does this pull request change what data or activity we track or use?


n/a

## Testing instructions:






- Make sure you have a simple site that points to your sandbox
- It should have a couple of posts
- To reproduce the issue, visit `/?random`: you should see the homepage.
- Apply this PR changes to your sandbox
  - Checkout this branch locally
  - Run `jetpack rsync mu-wpcom-plugin` to sync the changes with your sandbox
- Visit `/?random`
- You should see a random post
